tests/draft2020-12/unevaluatedProperties.json

```diff
@@ -809,5 +809,223 @@
+                "data": {
+                    "foo": "test",
+                    "bar": "test"
+                },
+                "valid": false
+            }
+        ]
+    },
+    {
+        "description": "property is evaluated in an uncle schema to unevaluatedProperties",
+        "comment": "see https://stackoverflow.com/questions/66936884/deeply-nested-unevaluatedproperties-and-their-expectations",
+        "schema": {
+            "type": "object",
+            "properties": {
+                "foo": {
+                    "type": "object",
+                    "properties": {
+                        "bar": {
+                            "type": "string"
+                        }
+                    },
+                    "unevaluatedProperties": false
+                  }
+            },
+            "anyOf": [
+                {
+                    "properties": {
+                        "foo": {
+                            "properties": {
+                                "faz": {
+                                    "type": "string"
+                                }
+                            }
+                        }
+                    }
+                }
+            ]
+        },
+        "tests": [
+            {
+                "description": "no extra properties",
+                "data": {
+                    "foo": {
+                        "bar": "test"
+                    }
+                },
+                "valid": true
+            },
+            {
+                "description": "uncle keyword evaluation is not significant",
+                "data": {
+                    "foo": {
+                        "bar": "test",
+                        "faz": "test"
+                    }
+                },
+                "valid": false
+            }
+        ]
+    },
+    {
+        "description": "property is evaluated in a nephew schema to unevaluatedProperties",
+        "schema": {
+            "type": "object",
+            "properties": {
+                "bar": {
+                    "type": "string"
+                }
+            },
+            "unevaluatedProperties": false,
+            "anyOf": [
+                {
+                    "properties": {
+                        "faz": {
+                            "type": "string"
+                        }
+                    }
+                }
+            ]
+        },
+        "tests": [
+            {
+                "description": "no extra properties",
+                "data": {
+                    "bar": "test"
+                },
+                "valid": true
+            },
+            {
+                "description": "all in-place applicators are evaluated before unevaluatedProperties",
+                "data": {
+                    "bar": "test",
+                    "faz": "test"
+                },
+                "valid": true
+            }
+        ]
+    },
```

Comment:
This also appears to be a duplicate of "unevaluatedProperties with nested properties" on line 157

Reply:
again it's pretty close; it should be safe to remove.